test/units/module_utils/facts/system/distribution/fixtures/rockylinux_8_3.json

```
@@ -0,0 +1,53 @@
+{
+    "name": "RockyLinux 8.3",
+    "distro": {
+        "codename": "",
+        "id": "rockylinux",
+        "name": "RockyLinux",
+        "version": "8.3",
+        "version_best": "8.3",
+        "lsb_release_info": {
+            "lsb_version": ":core-4.1-amd64:core-4.1-noarch",
+            "distributor_id": "RockyLinux",
+            "description": "RockyLinux release 8.3",
+            "release": "8.3",
+            "codename": ""
+        },
+        "os_release_info": {
+            "name": "RockyLinux",
+            "version": "8.3",
+            "id": "rockylinux",
+            "id_like": "rhel fedora",
```

Please match the json fixtures according to `/etc/os-release`. Rocky Linux is separated. id_like is `rhel fedora`
```suggestion
    "name": "Rocky Linux 8.3",
    "distro": {
        "codename": "",
        "id": "rocky",
        "name": "Rocky Linux",
        "version": "8.3",
        "version_best": "8.3",
        "lsb_release_info": {
            "lsb_version": ":core-4.1-amd64:core-4.1-noarch",
            "distributor_id": "Rocky",
            "description": "Rocky Linux release 8.3",
            "release": "8.3",
            "codename": ""
        },
        "os_release_info": {
            "name": "Rocky Linux",
            "version": "8.3",
            "id": "rocky",
            "id_like": "rhel fedora",
```

```
NAME="Rocky Linux"
VERSION="8"
ID="rocky"
ID_LIKE="rhel fedora"
VERSION_ID="8"
PLATFORM_ID="platform:el8"
PRETTY_NAME="Rocky Linux 8"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:rocky:rocky:8"
HOME_URL="https://rockylinux.org/"
BUG_REPORT_URL="https://bugs.rockylinux.org/"
ROCKY_SUPPORT_PRODUCT="Rocky Linux"
ROCKY_SUPPORT_PRODUCT_VERSION="8"
```